bridge/scripts/lib/madnetTasks.ts

```diff
@@ -217,3 +217,63 @@ task(
   const event = intrface.decodeEventLog("DepositReceived", data, topics);
   console.log(event);
 });
+
+task("scheduleMaintenance", "Calls schedule Maintenance").setAction(
+  async (taskArgs, hre) => {
+    const { ethers } = hre;
+    const iface = new ethers.utils.Interface([
+      "function scheduleMaintenance()",
+    ]);
+    const input = iface.encodeFunctionData("scheduleMaintenance", []);
+    console.log("input", input);
+    const [admin] = await ethers.getSigners();
+    const adminSigner = await ethers.getSigner(admin.address);
+    const factory = await ethers.getContractAt(
+      "MadnetFactory",
+      "0x0b1f9c2b7bed6db83295c7b5158e3806d67ec5bc"
+    );
+    const validatorPool = await hre.ethers.getContractAt(
+      "ValidatorPool",
+      await factory.lookup(
+        hre.ethers.utils.formatBytes32String("ValidatorPool")
+      )
+    );
+    await (
+      await factory
+        .connect(adminSigner)
+        .callAny(validatorPool.address, 0, input)
+    ).wait();
+  }
+);
+
+task(
+  "pauseEthdkgArbritaryHeigh",
+  "Forcing consensus to stop on block number defined by --input"
+)
+  .addParam("madnetHeight", "The block number after the latest block mined")
+  .setAction(async (taskArgs, hre) => {
+    const { ethers } = hre;
+    const iface = new ethers.utils.Interface([
+      "function pauseConsensusOnArbitraryHeight(uint256)",
+    ]);
+    const input = iface.encodeFunctionData("pauseConsensusOnArbitraryHeight", [
+      taskArgs.input,
+    ]);
+    const [admin] = await ethers.getSigners();
+    const adminSigner = await ethers.getSigner(admin.address);
+    const factory = await ethers.getContractAt(
+      "MadnetFactory",
+      "0x0b1f9c2b7bed6db83295c7b5158e3806d67ec5bc"
```

Comment:
Can we get the Madnet factory address in this case `0x0b1f9c2b7bed6db83295c7b5158e3806d67ec5bc` as a mandatory input variable?